### PR TITLE
Improved install docs (Vagrant, Homebrew, MAMP)

### PR DESCRIPTION
**Pull request by chillu**
Modified some offline instructions from @halkyon to get Homebrew going, but went the custom PHP binary route rather than mucking around with PECL. More details in commit messages.

@camfindlay Are you OK with adding the Bitnami reference?
@dhensby Are you OK with linking to your/BetterBrief's Vagrant installer?

### CODE REVIEW

File: docs/en/00_Getting_Started/01_Installation/04_Other_installation_Options/Mac_OSX_Homebrew.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+Now you can install Homebrew itself:
+
+	ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+
+## Install PHP
+
+We're installing PHP 5.5 here, with the required `mcrypt` module:
+
+	brew install php55 php55-mcrypt
```
Comment (halkyon) [inline docs/en/00_Getting_Started/01_Installation/04_Other_installation_Options/Mac_OSX_Homebrew.md]: Don't you have to setup some custom tap for the php packages with homebrew?

File: docs/en/00_Getting_Started/01_Installation/04_Other_installation_Options/Mac_OSX_Homebrew.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+If you run projects on different PHP versions, or care about additional PHP module support
+and other dependencies such as MariaDB, we recommend an installation through [Homebrew](http://brew.sh/).
+
+Check out the [MAC OSX with MAMP](../Mac_OSX) for an alternative installation process
+which packages up the whole environment into a convenient application.
+
+## Requirements
+
+Since we're compiling PHP, some build tooling is required.
+Download and install the free [Apple Xcode development environment](https://developer.apple.com/downloads).
```
Comment (halkyon) [inline docs/en/00_Getting_Started/01_Installation/04_Other_installation_Options/Mac_OSX_Homebrew.md]: This link should be `https://developer.apple.com/xcode/downloads/` instead. The current one just goes to a developer login page.

File: docs/en/00_Getting_Started/01_Installation/04_Other_installation_Options/Mac_OSX_Homebrew.md (hunk 1 above)
Comment (stevie-mayhew) [inline docs/en/00_Getting_Started/01_Installation/04_Other_installation_Options/Mac_OSX_Homebrew.md]: We run the following before installing php with brew:

```bash
brew cleanup
brew doctor
brew update
brew tap homebrew/dupes
brew tap josegonzalez/homebrew-php
brew tap --repair
```

File: docs/en/00_Getting_Started/01_Installation/04_Other_installation_Options/Mac_OSX_Homebrew.md (hunk 2 above)
Comment (stevie-mayhew) [inline docs/en/00_Getting_Started/01_Installation/04_Other_installation_Options/Mac_OSX_Homebrew.md]: I believe that `xcode-select --install` works without having to install XCode manually prior to installation?

File: docs/en/00_Getting_Started/01_Installation/04_Other_installation_Options/Mac_OSX_Homebrew.md (hunk 1 above)
Comment (chillu) [inline docs/en/00_Getting_Started/01_Installation/04_Other_installation_Options/Mac_OSX_Homebrew.md]: Good point, done

File: docs/en/00_Getting_Started/01_Installation/04_Other_installation_Options/Mac_OSX_Homebrew.md (hunk 2 above)
Comment (chillu) [inline docs/en/00_Getting_Started/01_Installation/04_Other_installation_Options/Mac_OSX_Homebrew.md]: Done

**Comment by chillu** [comment]
@stevie-mayhew Thanks for the feedback, I've linked to the troubleshooting docs for Homebrew. Good point on `xcode-select`, I've checked out a few tutorials around the web and appears that you're right. I've removed the download link.

**Comment by halkyon** [comment]
My beautiful IIS 6 docs are being removed... ;-)
Looks good to me.

**Comment by dhensby** [comment]
@chillu happy to have the vagrant repo linked to :)

I would go into more detail about why vagrant is so awesome (portable dev environments, very few local deps, shareable and identical environments for your team, "like live" environments)

**Comment by chillu** [comment]
@dhensby Awesome! Want to make a rewording suggestion for Vagrant? Is anything blocking the merge still?

**Pull request merged**